src/tests/components/pageNotFound.test.js

```diff
@@ -0,0 +1,31 @@
+
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { mount } from 'enzyme';
+import { MemoryRouter } from 'react-router';
+import Home from '../../components/Home';
+import PageNotFound from '../../components/404';
+import Routes from '../../routes/index';
+
+jest.mock("react-router-dom/BrowserRouter", () => {
+    return ({ children }) => <div>{children}</div>;
+  });
+
+test('invalid path should redirect to 404', () => {
+  const wrapper = mount(
+    <MemoryRouter initialEntries={[ '/jibugcfqhjbn' ]}>
+      <Routes/>
+    </MemoryRouter>
+  );
+  expect(wrapper.find(PageNotFound)).toHaveLength(1);
+});
+
+test('valid path should not redirect to 404', () => {
+  const wrapper = mount(
+    <MemoryRouter initialEntries={[ '/' ]}>
+      <Routes/>
+    </MemoryRouter>
+  );
+  expect(wrapper.find(Home)).toHaveLength(1);
+  expect(wrapper.find(PageNotFound)).toHaveLength(0);
+});
```

Comment:
Please add EOF line

Reply:
I have added an extra line at the end, thanks for catching it out.